src/queue.js

```diff
@@ -1,5 +1,28 @@
-class Queue {
+'use strict';
 
+class Queue {
+    constructor() {
+        // emoty queue
+        this.queue = [];
+    }
+    
+    enqueue(item) {
+        // add item to the queue
+        this.queue.unshift(item);
+    }
+    
+    dequeue() {
+        // remove item only if there's something to remove
+        if (!this.size() <= 0) {
```

Comment:
The `!` operator here is not necessary since you're also checking that `this.size() <= 0`.